scapy/contrib/automotive/uds_scan.py

```diff
@@ -573,14 +751,14 @@ class UDS_RCEnumerator(UDS_Enumerator):
 
     def _get_initial_requests(self, **kwargs):
         # type: (Any) -> Iterable[Packet]
+        type_list = kwargs.pop("type_list", [1, 2, 3])
         scan_range = kwargs.pop("scan_range", range(0x10000))
-        return itertools.chain(
-            (UDS() / UDS_RC(routineControlType=1, routineIdentifier=x)
-             for x in scan_range),
-            (UDS() / UDS_RC(routineControlType=2, routineIdentifier=x)
-             for x in scan_range),
-            (UDS() / UDS_RC(routineControlType=3, routineIdentifier=x)
-             for x in scan_range))
+
+        return \
```

Comment:
I personally don't like this breaks after return

Reply:
Changed